### PR TITLE
[29335] Remove legacy action menus

### PR DESCRIPTION
**Pull request by HDinger**
Remove legacy action menu implementation

https://community.openproject.com/projects/openproject/work_packages/29335/activity

### CODE REVIEW

File: app/views/workflows/copy.html.erb [hunk 1]
```diff
@@ -27,7 +27,7 @@ See docs/COPYRIGHT.rdoc for more details.
 
 ++#%>
 <% html_title l(:label_administration), l(:label_workflow_plural) -%>
-<%= render partial: 'action_menu', locals: { title: Workflow.model_name.human } %>
+<%= render partial: 'toobar', locals: {title: Workflow.model_name.human } %>
```
Comment (oliverguenther) [inline app/views/workflows/copy.html.erb]: These should read 'toolbar'. It's only a typo and identical in the partial name (so it works), but it bothers me! :D

File: app/views/workflows/edit.html.erb [hunk 2]
```diff
@@ -27,8 +27,8 @@ See docs/COPYRIGHT.rdoc for more details.
 
 ++#%>
 <% html_title l(:label_administration), l(:label_workflow_plural) -%>
-<%= render partial: 'action_menu', locals: { title: Workflow.model_name.human } %>
-<%= render partial: 'layouts/action_menu_specific' %>
+<%= render partial: 'toobar', locals: {title: Workflow.model_name.human } %>
```
Comment (oliverguenther) [inline app/views/workflows/edit.html.erb]: Same as above

File: app/views/workflows/index.html.erb [hunk 3]
```diff
@@ -26,7 +26,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See docs/COPYRIGHT.rdoc for more details.
 
 ++#%>
-<%= render partial: 'action_menu', locals: { title: l(:label_workflow_plural) } %>
+<%= render partial: 'toobar', locals: {title: l(:label_workflow_plural) } %>
```
Comment (oliverguenther) [inline app/views/workflows/index.html.erb]: Same as above

**Review by oliverguenther — CHANGES_REQUESTED** [review]

File: app/views/workflows/copy.html.erb (hunk 1 above)
Comment (HDinger) [reply app/views/workflows/copy.html.erb]: Ah, good catch 👍

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**